wallet-new/server/Cardano/Wallet/Server.hs

```diff
@@ -19,12 +21,15 @@ import           Servant
 -- time of writing) two ways of accessing the V0 API, which are either by hitting
 -- `/api/..` or by hitting `/api/v0/...`.
 
--- walletServer :: (HasConfigurations, HasCompileInfo) => Server WalletAPI
-walletServer
-    :: forall ctx m.
-       ( MonadFullWalletWebMode ctx m )
-    => (forall a. m a -> Handler a)
-    -> Server WalletAPI
+walletServer :: ( Migration.HasConfiguration
+                , Migration.HasCompileInfo
+                , Migration.HasUpdateConfiguration
+                , Migration.HasSscConfiguration
+                , Migration.HasNodeConfiguration
+                , Migration.HasInfraConfiguration
```

Comment:
We combined type constraint `HasConfigurations` for this. It's in `cardano-sl`, module `Pos.Launcher.Configuration`.

Reply:
Oh, neat, I had no clue we had such a thing!